service/internal/server/server.go

```diff
@@ -104,13 +101,19 @@ type CORSConfig struct {
 	MaxAge           int      `mapstructure:"maxage" json:"maxage" default:"3600"`
 }
 
```

Comment:
```suggestion
const GRPCGatewayDeprecationDate = fmt.Sprintf("@%d", time.Date(2025, time.March, 25, 0, 0, 0, 0, time.UTC).Unix())

```

Reply:
You can't do a constant like this in go.